android/src/main/kotlin/com/mapbox/maps/mapbox_maps/MapboxMapController.kt

```diff
@@ -202,6 +204,21 @@ class MapboxMapController(
         dispose()
         result.success(null)
       }
+      "map#snapshot" -> {
+        val mapView = mapView
+        val snapshot = mapView?.snapshot()
+        if (mapView == null) {
+          result.error("2342345", "Failed to create snapshot: map view is not found.", null)
```

Comment:
what is the meaning of this error code? do we have a list of error codes somewhere we can look up?

Reply:
I wasn't sure what to put there, so just randomly mashed my fingers against the number row 🙈